### PR TITLE
#1 - project setup

### LINKED ISSUE #1: Project setup
Let's setup project and add .gitattributes, .gitignore, .pdd, .rultor.yml and pom.xml. Also readme should be filled with badges and repository information.

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #1
I've set up the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+[![Hits-of-Code](https://hitsofcode.com/github/artipie/rpm-adapter)](https://hitsofcode.com/view/github/artipie/rpm-adapter)
+[![Maven Central](https://img.shields.io/maven-central/v/com.artipie/rpm-adapter.svg)](https://maven-badges.herokuapp.com/maven-central/com.artipie/rpm-adapter)
+[![PDD status](http://www.0pdd.com/svg?name=artipie/rpm-adapter)](http://www.0pdd.com/p?name=artipie/rpm-adapter)
+
+This Java library turns your binary storage
+(files, S3 objects, anything) into a Debian repository.
+You may add it to your binary storage and it will become
+a fully-functionable Debian repository, which
+[`apt`](https://ru.wikipedia.org/wiki/Advanced_Packaging_Tool) 
+will perfectly understand.
```
Comment (genryxy) [inline README.md:19]: @olenagerasimova it seems better to add a link to the english version

**Comment by olenagerasimova** [comment]
@genryxy thanks, corrected

**Comment by olenagerasimova** [comment]
@olegmoz could you please take a look and merge?

File: .rultor.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+docker:
+  image: g4s8/rultor-jdk11:alpine3.10
```
Comment (olegmoz) [inline .rultor.yml:2]: @olenagerasimova I think this project is for Java8, so it should not use Java 11 image

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    </developer>
+  </developers>
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/artipie/debian-adapter/issue</url>
+  </issueManagement>
+  <properties>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+  </properties>
```
Comment (olegmoz) [inline pom.xml:64]: @olenagerasimova this project should be compiled for Java8

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+  <version>1.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  <name>debian-adapter</name>
+  <description>Debian adapter</description>
+  <url>https://github.com/artipie/debian-adapter</url>
+  <inceptionYear>2020</inceptionYear>
+  <licenses>
+    <license>
+      <name>MIT</name>
+      <url>https://github.com/artipie/debian-adapter/blob/main/LICENSE.txt</url>
```
Comment (olegmoz) [inline pom.xml:42]: @olenagerasimova license is now stored in `LICENSE`, not `LICENSE.txt`. I think URL here should stay as it is, and `LICENSE` file should be renamed

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+    </dependency>
+    <dependency>
+      <groupId>com.jcabi</groupId>
+      <artifactId>jcabi-log</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.github.jknack</groupId>
+      <artifactId>handlebars</artifactId>
+      <version>4.2.0</version>
+    </dependency>
```
Comment (olegmoz) [inline pom.xml:79]: @olenagerasimova not sure why we may need `handlebars` in this project

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+  <properties>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>com.amihaiemil.web</groupId>
+      <artifactId>eo-yaml</artifactId>
+      <version>5.1.5</version>
+    </dependency>
```
Comment (olegmoz) [inline pom.xml:70]: @olenagerasimova not sure that `eo-yaml` dependency is useful in this project

File: pom.xml [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+      <groupId>com.github.jknack</groupId>
+      <artifactId>handlebars</artifactId>
+      <version>4.2.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.jcabi</groupId>
+      <artifactId>jcabi-matchers</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <!-- maven -->
```
Comment (olegmoz) [inline pom.xml:85]: @olenagerasimova `<!-- maven -->` comment does not make sense to me

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see some comments

File: .rultor.yml (hunk 2 above)
Comment (olenagerasimova) [reply .rultor.yml:2]: @olegmoz that's rultor docker image, not related to the project

**Comment by olenagerasimova** [comment]
@olegmoz thanks, fixed and answered

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, good to merge

**Pull request merged**